test/functional/rpc_users.py

```diff
@@ -84,6 +94,39 @@ def test_auth(self, node, user, password):
         self.log.info('Wrong...')
         assert_equal(401, call_with_auth(node, user + 'wrong', password + 'wrong').status)
 
+    def test_rpccookieperms(self):
+
+        if os.name == 'nt':
+            raise SkipTest(f"Skip cookie file permissions checks as OS detected as: {os.name=:}")
+        self.log.info('Check cookie file permissions can be set using -rpccookieperms')
+
+        cookie_file_path = self.nodes[1].chain_path / '.cookie'
+
+        def test_perm(perm: Optional[str]):
+            if not perm:
+                perm = f"{(DEFAULT_COOKIE_PERMISSION & PERM_BITS_UMASK):03o}"
+                self.restart_node(1)
+            else:
+                self.restart_node(1, extra_args=[f"-rpccookieperms={perm}"])
+            file_stat = os.stat(cookie_file_path)
+            actual_perms = f"{(file_stat.st_mode & PERM_BITS_UMASK):03o}"
+            assert_equal(perm, actual_perms)
+
+       # Remove any leftover rpc{user|password} config options from previous tests
```

Comment:
Missing a space in the indentation

Reply:
Thanks! My formatter is having some difficulty with diff/range-based formatting at the moment. Fixed.